src/javascript/app/pages/user/new_account/virtual_acc_opening.js

```diff
@@ -107,6 +124,7 @@ const VirtualAccOpening = (() => {
             const residence   = response.echo_req.residence;
             Client.set('residence', residence, new_account.client_id);
             LocalStore.remove('gclid');
+            LocalStore.remove('date_first_contact');
```

Comment:
I'm not sure if this is the right place to do it (I wanted to remove `date_first_contact` LocalStorage item immediately after successful login).

Reply:
I don't think we need it here, if we have it on response of authorize it should handle this case as well. it needs to be tested.

Reply:
it should be on line 131 then